test/lnd_services_mock.go

```diff
@@ -122,6 +122,8 @@ type LndMockServices struct {
 
 	Height int32
 
+	RelevantTransactions []*wire.MsgTx
```

Comment:
Is the word `Relevant` accurate in the mock context? I'd expect something like `AllTransactions`

Reply:
Fixed.